guidelines/design-document-template.md

```diff
@@ -0,0 +1,35 @@
+# Design Document - My proposal 
+
+Author : **myemail@email.com**
+Date : Oct 2020
+
+## Overview 
+
+### Motivation
+Motivation to have this design document 
+
+### Examples &  Prior Art 
+Examples or references in how other solved similar or same problems
+
+```
+Hi,{mf}!
+```
+
+
+## Design 
+
+Discuss any background and motivation not already in the summary above. Give implementation details and all posible alternatives to solve the problem including affectations to other components.
```

Comment:
```suggestion
Discuss any background and motivation not already in the summary above. Give implementation details and possible alternatives to solve the problem including effects on other components.
```

Explicitly requiring consideration of _all_ alternatives is rather a lot to ask for.

Reply:
> implementation details

They don't belong in a design document.
Behavior does.